ansible/configs/sap-integration/software.yml

```diff
@@ -0,0 +1,209 @@
+---
+- name: Step 004 - Software
+  hosts: bastions
+  gather_facts: false
+  become: true
+  tasks:
+    - name: Debug ocp4-cluster software playbook
+      debug:
+        msg: "ocp4-cluster Software playbook"
+
+- name: Ensure NFS Server is installed, SAP Software Device Mounted and Ansible Installed
+  hosts: bastions
+  become: True
+  gather_facts: True
+  tasks:
+
+    - name: Ensure NFS directory exists
+      file:
+        path: "/nfs"
+        state: directory
+
+    - name: Mount up device by UUID
+      mount:
+        path: /nfs
+        src: /dev/vdb
+        fstype: xfs
+        state: present
+
+    - name: 'Ensure required packages are installed'
+      package:
+        name: '{{ item }}'
+        state: installed
+      with_items:
+      - nfs-utils
+      - firewalld
+
+    - name: 'Ensure firewalld is running'
+      service:
+        name: firewalld
+        state: started
+        enabled: yes
+
+    - name: 'Open Firewall for NFS use'
+      firewalld:
+        port: "{{ item }}"
+        permanent: yes
+        state: enabled
+        immediate: yes
+      with_items:
+      - 111/tcp
+      - 111/udp
+      - 2049/tcp
+      - 2049/udp
+
+    - name: "Ensure export file contains the directory to be shared"
+      lineinfile:
+        path: /etc/exports
+        state: present
+        regexp: "^/nfs"
+        line: "/nfs *(insecure,rw,no_root_squash,no_wdelay,sync)"
+
+    - name: 'Ensure nfs-server is restarted and running'
+      service:
+        name: nfs-server
+        state: restarted
+        enabled: yes
+
+    - name: 'Ensure SFLIGHT export information can be read by anyone'
+      file:
+        path: /nfs/SFLIGHT/index
+        state: directory
+        recurse: yes
+        owner: root
+        group: root
+        mode: 0755
+
+    - name: Ensure /nfs directory is owned by 'cloud-user'
+      file:
+        path: /nfs
+        state: directory
+        owner: cloud-user
+
+    - name: Ensure SAP software is not present for external (non Red Hatters) requesters
+      file:
+        state: absent
+        path: "{{ item }}"
+      loop:
+        - '/nfs/HANA_installation/'
+        - '/nfs/S4HANA_installation/'
+        - '/nfs/SAPCAR/'
+        - '/nfs/SAPHOSTAGENT/'
+      when: not rh_internal
+
+- name: Ensure NFS share is mounted in HANA and S/4HANA Hosts
+  hosts: hanas:s4hanas
+  become: True
+  gather_facts: True
+  tasks:
+
+    - name: Ensure Software directory exists
+      file:
+        path: "/software"
+        state: directory
+
+    - name: Ensure NFS share is mounted
+      mount:
+        path: "/software"
+        src: "bastion-{{ guid }}:/nfs"
+        fstype: nfs4
+        opts: "rw,bg,hard,_netdev"
+        state: mounted
+
+- name: Ensure required repos for SAP are enabled
```

Comment:
Why are you switching to a new play here? you're targetting the same hosts with the same privilege.

Reply:
done